examples/pytorch/gat/ray_util.py

```diff
@@ -0,0 +1,150 @@
+"""Baseline Ray Trainer."""
+from typing import Dict
+import os
+import platform
+import numpy as np
+import torch
+import ray
+import ray.train as train
+import horovod.torch as hvd
+import ray.train.torch
+from ray.train.horovod import HorovodTrainer
+from ray.air import session
+from ray.air.config import ScalingConfig
+from deepgnn import TrainMode, get_logger
+from deepgnn.graph_engine import create_backend, BackendOptions
+from deepgnn.graph_engine.samplers import GENodeSampler, GEEdgeSampler
+from deepgnn.pytorch.common import get_args
+from deepgnn.pytorch.common.consts import PREFIX_CHECKPOINT
+from deepgnn.pytorch.common.utils import rotate_checkpoints, get_sorted_checkpoints
+
+
+def train_func(config: Dict):
```

Comment:
Why do we have almost identical ray_util module across most of the examples? Please extract common functionality.

Reply:
This is what the migrate script generates and replaces run_dist with, I will refactor the examples and add some helper functions to reduce code.

Reply:
Updated and removed repeated checkpoint code.